File: grassseed.py
c=float(input())
if(0<c<=100.0):
  l=int(input())
  if(1<=l<=100):
    sum=0.0
    for i in range(0,l):
      lw=input().split()
      sum+=float(lw[0])*float(lw[1])
    print(sum*c)

File: grassseed.rb
c=gets.to_f
if(0<c and c<=100)
  l=gets.to_i;
  if(0<l and 1<=100)
    sum=0.0;
    for i in 1..l
      w,h=gets.chomp.split.map(&:to_f)
      sum+=w*h
    end
  end
  puts sum*c
end